insights/parsers/uname.py

```diff
@@ -209,7 +210,11 @@ def parse_content(self, content):
             raise UnameError('Empty uname line', '')
 
         data = {}
-        uname_line = content[-1]  # read the last line instead of the first
+        uname_line = None
+        for line in content:
```

Comment:
@wushiqinlou - for tarball collected `foreman-debug`, the uname's output looks like

~~~
COMMAND> uname -a

Linux slolmeprdrhsv01.ipa.lme.co.uk 3.10.0-327.10.1.el7.x86_64 #1 SMP Sat Jan 23 04:54:55 EST 2016 x86_64 x86_64 x86_64 GNU/Linux
~~~

So, you'd better get the line that starts with "Linux" as the `uname_line`.

BTW, you can implement it in a list comprehension.